runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -888,12 +891,12 @@ You can find help on just about any subject, by giving an argument to the
 ~~~
 # Lesson 7.2: CREATE A STARTUP SCRIPT
 
-** Enable Vim features. **
+** Enable Neovim features. **
 
-Vim has many more features than Vi, but most of them are disabled by
-default. To start using more features you have to create a "vimrc" file.
+Neovim has many more features than Vi, but most of them are disabled by default.
+To start using more features you have to create a "init.vim" or "init.lua" file.
```

Comment:
We only refer to `init.vim` in the following; maybe keep it to that here as well. (Don't introduce complexity _without necessity_.)

Reply:
This one feels like when you were in school in physics and every year starts out "We taught you that x is a fact, well actually that's not true, it's sometimes true but here's the truer true"

That being said I agree with your point and have rectified it the latest commit

Reply:
Maybe then say "To set up your configuration, you _can_ create..." so it's not wrong?